.github/workflows/python-package.yml

```diff
@@ -28,4 +28,8 @@ jobs:
         flake8 . --count --max-complexity=10 --max-line-length=127 --statistics
     - name: Test with pytest
       run: |
-        echo Add your testing here
+        pytest -rp
+    - name: Publish the egg
+      run: |
+        cp .pypirc $HOME/
```

Comment:
Nice!
Could also be done using the [twine](https://pypi.org/project/twine/) package